CalendarApp/GoogleCalendarService/Manager/EventManager.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Calendar.v3.Data;
+using Google.Apis.Requests;
+using GoogleCalendarService.Extensions;
+
+namespace GoogleCalendarService.Manager
+{
+    public class EventManager
+    {        
+        private readonly CalendarService _service;
+
+        public EventManager(CalendarService service)
+        {
+            _service = service;
+        }
+
+        //Need this for UpdateEventMethod
+        public List<Event> CheckEvent(string calendarId, Event eventForAdd, bool forcedEvent, string updateEventId)
+        {
+            eventForAdd.Id = updateEventId;
+            List<Event> conflictedEvents = CheckEvent(calendarId, eventForAdd, forcedEvent);
+            return conflictedEvents;
+        }
+
+        public List<Event> CheckEvent(string calendarId, Event eventForAdd, bool forcedEvent)
+        {
+            IList<Event> eventList = new List<Event>();
+            List<Event> conflictedEvents = new List<Event>();
+            if (forcedEvent)
+                return conflictedEvents;
+
+            EventPeriodicityHolder eventForAddHolder = new EventPeriodicityHolder(eventForAdd);
+
+            Events request = _service.Events.List(calendarId).Execute();
+            string token = request.NextPageToken;
+            while (token !=null)
+            {
+                var tempRequst = _service.Events.List(calendarId);
+                tempRequst.PageToken = token;
+                eventList = eventList.Concat(tempRequst.Execute().Items).ToList();
+                var updatedRequest = tempRequst.Execute();
+                token = updatedRequest.NextPageToken;
+            }
+
+            List<Event> allExistedEvents = new List<Event>();
+
+            foreach (var ev in eventList)
+            {
+                EventPeriodicityHolder temp = new EventPeriodicityHolder(ev);
+                allExistedEvents = allExistedEvents.Concat(temp.EventList).ToList();
+            }
+            foreach (var ev in eventForAddHolder.EventList)
+            {
+                conflictedEvents = conflictedEvents.Concat(FindConflictedEvents(allExistedEvents, ev)).ToList();
+            }
+            return conflictedEvents.GroupBy(x => x.Id).Select(x => x.First()).ToList();
+        }
+
+        private List<Event> FindConflictedEvents(List<Event> eventList,Event eventForAdd )
```

Comment:
can you add some spaces between parameters and remove extra spaces at the and of parameters list